insolar/payload/payload.go

```diff
@@ -431,6 +439,14 @@ func Unmarshal(data []byte) (Payload, error) {
 		pl := GetRequest{}
 		err := pl.Unmarshal(data)
 		return &pl, err
+	case TypeGetPulse:
+		pl := GetPulse{}
+		err := pl.Unmarshal(data)
+		return &pl, err
+	case TypePulse:
+		pl := Pulse{}
+		err := pl.Unmarshal(data)
+		return &pl, err
```

Comment:
you injected these two right in the middle of group "GetRequest", "GetRequestInfo" and "TypeRequest"